CHANGELOG.rst

```diff
@@ -4,7 +4,7 @@ Releases
 0.3.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Added an option to disable "required/optional" strictness on structs.
```

Comment:
@breerly I'm anticipating you'll tell me this belongs in 0.4.0 :)

I could also put "print as JSON" in there.